partner_interconnect_attachments.go

```diff
@@ -46,6 +46,38 @@ type PartnerInterconnectAttachmentCreateRequest struct {
 	BGP BGP `json:"bgp,omitempty"`
 }
 
+// PartnerInterconnectAttachmentCreateRequest represents a request to create a Partner Interconnect Attachment.
+type PartnerInterconnectAttachmentRequestBody struct {
```

Comment:
Since this is a struct used to work around a breaking change, I don't think it's ever intended to be used by external consumers of this library, and therefore shouldn't be a public struct:
```suggestion
type partnerInterconnectAttachmentRequestBody struct {
```